python/sdk/test/batch/model/env.yaml

```diff
@@ -1,9 +1,9 @@
 dependencies:
   - pip:
       - mlflow==1.23.0
-      - scikit-learn==1.0.1  # >=1.1.2 upon python 3.7 deprecation
+      - scikit-learn==1.0.2  # >=1.1.2 upon python 3.7 deprecation
       - joblib>=0.13.0,<1.2.0  # >=1.2.0 upon upgrade of kserve's version
-      - numpy==1.19.5 # https://github.com/Azure/MachineLearningNotebooks/issues/1314
+      - numpy>=1.19.5 # https://github.com/Azure/MachineLearningNotebooks/issues/1314
```

Comment:
For test dependencies, it's better to pin it to the working version to avoid test regression when new version coming out.